example/build.gradle

```diff
@@ -6,16 +6,16 @@ repositories {
 
 dependencies {
     compile project(':stripe')
-    compile 'com.android.support:support-v4:23.4.0'
+    compile 'com.android.support:support-v4:24.1.1'
     androidTestCompile 'junit:junit:4.12'
-    androidTestCompile 'org.robolectric:robolectric:3.0'
+    androidTestCompile 'org.robolectric:robolectric:3.1.4'
```

Comment:
There are no more tests in the example project, so there's no need to add test compile variables to it.